src/components/SiteProfileComponents/SiteProfile.tsx

```diff
@@ -283,10 +284,12 @@ class SiteProfile extends React.Component {
     if(pageNumber == 1)
     {
```

Comment:
I'm a little worried these numbers are super opaque and won't make sense later. can we at least replace these with named constants?

Reply:
Changes have been made sir!